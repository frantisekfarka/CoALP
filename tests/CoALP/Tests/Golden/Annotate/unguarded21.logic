q(s(f(s(X1),X2))) :- q(s(f(X2,X2))).
