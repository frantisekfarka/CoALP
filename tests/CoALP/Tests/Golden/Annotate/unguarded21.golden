q(s(f(s(V_2), V_3)), transform-func-1(V_-4)) :- q(s(f(V_3, V_3)), V_-4).
